### DISCUSSION TITLE
has anybody know cortx gui  username and password?

### DISCUSSION THREAD

**Discussion by deelaaay** [Seagate/cortx#1400] [General]
the cluster env build and deploy from branch 2.0.0-559。https://local-ip:28100/#/preboarding/login   . i had used cortxadmin/Cortxadmin@123 ,but it not useful.....

**Comment by mukul-seagate11** [comment]
> the cluster env build and deploy from branch 2.0.0-559。https://local-ip:28100/#/preboarding/login . i had used cortxadmin/Cortxadmin@123 ,but it not useful.....

yes correct but it was in previous CORTX design and now the architecture has moved to kubernetes

**Comment by mukul-seagate11** [comment]
So, let us know if you wanted to refer K8s OVA then you can refer https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst but CORTX UI is removed where IO performance can now be tested using CLI
